### ISSUE TITLE
Don't try to search for project file for the user

### ISSUE THREAD

**Issue by raph-amiard** [AdaCore/ada_language_server#23]
IMHO the ALS shouldn't try to find the gpr recursively for the user. If there is an error in the user settings it would be better to fail and communicate that to the user somehow.

As discussed in https://github.com/AdaCore/ada_language_server/pull/22_

**Comment by godunko** [comment]
I agree. ALS should report configuration error on first useful request/notification with detailed information when available. It is user/IDE responsibility to configure ALS properly.
